src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i of actions) {
+    if (i.type === 'addProperties') {
+      for (const key in i.extraData) {
+        state[key] = i.extraData[key];
+      }
+    }
+
+    if (i.type === 'removeProperties') {
+      for (const key of i.keysToRemove) {
+        delete state[key];
+      }
+    }
+
```

Comment:
And if you want, use a `switch`. 
![image](https://user-images.githubusercontent.com/108505176/183458933-86337825-cff2-4428-8076-cc249cd5b77f.png)

Reply:
agree

Reply:
Ok)

Reply:
Ok)